app/static/common.style.css

```diff
@@ -101,3 +101,47 @@ body {
   float: left;
   width: calc(100% - 255px);
 }
+
+.error-dialogue {
```

Comment:
Yikes, while looking through the style names here to compare with the Interactive Label, I noticed how badly mixed the names are there: https://github.com/ACMILabs/interactive-label/blob/main/app/static/style.css

Reply:
@sighmon Might sneak this in as a stretch goal for the Step 2 of the 1.0.